### ISSUE TITLE
installation error - chex incompatible with jax version

### ISSUE THREAD

**Issue by enozkan** [YoshitakaMo/localcolabfold#149]
During running the installation script, I get the following error:

```
Installing collected packages: biopython, jax
  Attempting uninstall: biopython
    Found existing installation: biopython 1.81
    Uninstalling biopython-1.81:
      Successfully uninstalled biopython-1.81
  Attempting uninstall: jax
    Found existing installation: jax 0.4.8
    Uninstalling jax-0.4.8:
      Successfully uninstalled jax-0.4.8
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
chex 0.1.7 requires jax>=0.4.6, but you have jax 0.3.25 which is incompatible.
Successfully installed biopython-1.79 jax-0.3.25
```
but the installation completes.

When I try to run a job, it appears that colabfold fails to find a gpu, which may be related to the error.
```
2023-04-16 10:26:09,371 Running colabfold 1.5.2 (09ba35598fb7224180a5c0336cd3d378a33817e1)
2023-04-16 10:26:09.433375: W external/org_tensorflow/tensorflow/tsl/platform/default/dso_loader.cc:66] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /software/cuda-11.5-el8-x86_64/extras/CUPTI/lib64:/software/cuda-11.5-el8-x86_64/lib64:/software/slurm-current-el8-x86_64/lib
2023-04-16 10:26:09.433411: W external/org_tensorflow/tensorflow/compiler/xla/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: UNKNOWN ERROR (303)
2023-04-16 10:26:09,433 WARNING: no GPU detected, will be using CPU
```
Not sure if the two are related and other software has no problem using the gpu.

**Comment by YoshitakaMo** [comment]
I guess the installation error is not related to the failure of GPU detection. Try executing `nvidia-smi` before `colabfold_batch` command to make sure the calculation node can detect a GPU. If `nvidia-smi` returns an error message `NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver.`, the cause is the machine environment.

**Comment by YoshitakaMo** [comment]
Or, the cause may be `Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory;`. Please try to fix this error at first.

**Comment by hseara** [comment]
`chex` issue should be solved by my pull request #151
